bottlecap/src/traces/proxy_aggregator.rs

```diff
@@ -30,4 +30,9 @@ impl Aggregator {
     pub fn get_batch(&mut self) -> Vec<ProxyRequest> {
         std::mem::take(&mut self.queue)
     }
+
+    /// Flush the queue.
+    pub fn flush(&mut self) {
```

Comment:
Should this be `clear`? That way we don't mix the concept of `flushing` with our `Flusher` class?

Reply:
```suggestion
    pub fn clear(&mut self) {
```